pagopa/pn-frontend
# Manage 451 error status code - pn-4285

## Short description
Show a custom error message when exchangeToken returns a 451 error status code

## List of changes proposed in this pull request
- Added in user.utility.ts the case 451
- Added new class for the new error
- Added error localization
- Added 451 case in tests

## How to test
For pa, pf and pg try to login with a tax outside the white list and check that user cannot login and the correct message is shown; for test I excluded DSRDNI00A01A225I (dino - pf) and WZCCLV58R49B051K (m.messina - pa)
